KeyboardKit/KeyboardAction.swift

```diff
@@ -0,0 +1,136 @@
+public enum KeyboardAction {
+    /// An action to cancelling an in-progress task or dismiss a prompt, consisting of the Escape (⎋) key and no modifiers.
+    case cancel
+    
+    /// A close action, consisting of the Escape (⎋) key and no modifiers.
```

Comment:
This is the input for cancel other than close. I’m fixing this up in a followup.